src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
You can remove the `else` keyword because the `if` block contains a `return` statement. This will simplify the control flow as suggested by checklist items 4 and 5.